app/ledger.js

```diff
@@ -209,18 +209,8 @@ if (ipc) {
   })
 
   ipc.on(messages.CHANGE_SETTING, (event, key, value) => {
-    if (!client) return
-
-    if (key === settings.PAYMENTS_CONTRIBUTION_AMOUNT) {
-      var amount = parseInt(value, 10)
-      var bravery = client.getBraveryProperties()
-
-      if (isNaN(amount) || (amount <= 0)) return
-
-      underscore.extend(bravery.fee, { amount: amount })
-      client.setBraveryProperties(bravery, callback)
-      getPaymentInfo()
-    }
+    if (key === settings.PAYMENTS_ENABLED) return initialize(value)
```

Comment:
these lines in app/index.js should be removed, right?
```
    ipcMain.on(messages.LEDGER_ENABLE, (e, enabled) => {
      ledger.enable(enabled)
    })
```

otherwise